docs/operating/transaction-gas.md

```diff
@@ -0,0 +1,90 @@
+
+### Pass on gas savings
+
+Because the Neon EVM operates on Solana, not Ethereum it takes advantage of Solana’s inexpensive transaction costs and favourable approach to only charge for storage allocation. This contrasts with Ethereum, where calculations require much more gas, and gas is charged for every change in data stored in Ethereum. As a result, gas usage on Solana is much cheaper than on Ethereum, and the Neon EVM passes these savings on to users.
+
+
+## Gas price: the Operator fee
+
+The Neon Proxy obtains the current prices of SOL and NEON tokens from the [pyth.network](http://pyth.network) oracles.
+
+Gas price = $SOL / $NEON * (1 + `PRX_OPERATOR_FEE`)
+
+> This ensures that the Neon Operator receives enough NEONs to cover the transaction cost in SOLs.
+
+The Neon Operator configures the value of `PRX_OPERATOR_FEE` where 1.0 represents 100% of the potential fee extraction, i.e. PRX_OPERATOR_FEE = "1.0”  
+
+For example:
+
+- $NEON = $0.25
+- $SOL = $10
+- PRX_OPERATOR_FEE = "1"
+- Gas price = 10 / 0.25 * (1 + 1) = 80 Galans
+
+Neon recommends that Neon Operators should initially set `PRX_OPERATOR_FEE` to “1.0” for Mainnet launch. This allows Operators to cover their hardware costs while transaction demands are low. As demand grows, Operators may adjust their fees in response.
+
+> For example, `PRX_OPERATOR_FEE` = “0.1” → 10% of the total possible Neon Operator fee.
+
+## Minimum gas price
+
+An Operator has two options:
+
+- **Recommended**: set up the `PRX_MINIMAL_GAS_PRICE` to 1 as per the default value in EVMs
+- Set up the minimal gas price `PRX_MINIMAL_GAS_PRICE` to reject low offers 
+
+> For example, by setting the gas price minimum to 65 Galans, Neon Proxy rejects transactions with a gas price of less than 65.
+
+If a transaction request arrives with a gas price below that of the Operator costs + Operator fee, but with a gas price larger than the designated `PRX_MINIMAL_GAS_PRICE`, the Neon Proxy accepts the transaction into mempool. 
+
+The Neon transaction will only be executed when the gas price covers Operator costs + Operator fee. If the number of transactions in the mempool grows to the `PRX_MEMPOOL_CAPACITY` (by default =4096), Neon Proxy removes the transaction with the smallest gas price from the mempool to accept transactions with the higher gas price. Therefore, it's not critical to set the `PRX_MINIMAL_GAS_PRICE` to a small value. 
+
+
+## Calculation configuration
+
+The Neon Proxy has several settings accommodate the calculation of gas-price on your required network:
+
+- `PRX_PP_SOLANA_URL`: the URL address of the Solana network with the main Pyth oracle contract that supplies $NEON and $SOL prices
```

Comment:
@afalaleev I belive there is more user support needed for line 74 here -- as the comment in Notion was this: In this case, the developer should configure the SOLANA_URL to the local Solana node, but PP_SOLANA_URL to the devnet (or mainnet) Solana node.

Reply:
If I unpack that -- you are saying that IF they are running Proxy Local, then there is a mixed bag, you need to 
- point at local Solana node AND 
- point at the deployed price feed contract on Devnet or Mainnet

BUT -- I only see one variable, so not sure how you control those two properties with one setting
Please elaborate